packages/@webex/internal-plugin-mercury/test/unit/spec/mercury-events.js

```diff
@@ -147,7 +142,7 @@ describe('plugin-mercury', () => {
         });
       });
 
-      describe('when `mercury.buffer_state` is received', () => {
+      /* describe('when `mercury.buffer_state` is received', () => {
```

Comment:
it's better to use `.skip()` rather then commenting out, because then at least the test is still part of the build and also it makes it very easy to find all skipped tests and it's clear in the test reports how many tests are skipped. Commenting out the code hides it all completely and it's not easy to find commented out tests